package.json

```diff
@@ -14,9 +14,11 @@
     },
     "homepage": "https://github.com/prezly/uploadcare#readme",
     "scripts": {
-        "build": "tsc --project .",
+        "build:main": "tsc --project .",
+        "build:umd": "webpack --output-filename index.js",
```

Comment:
cc: @oleg-semyonov , FYI

Let's not be selective about passing some webpack configuration in webpack config files and some via CLI.
Before this PR, all webpack config was in `webpack.config.js` - let's keep it that way.
Webpack config file can export an array of configs - each of them will produce a different build.

Then we will not need `npm-run-all` at all.
However, if you're worried about performance, we can use https://github.com/trivago/parallel-webpack instead. 

I don't want you to change anything right now, but in case there is one more config attribute to be passed via CLI - let's go back to webpack config file **only**.